dhis-2/dhis-api/src/main/java/org/hisp/dhis/security/Authorities.java

```diff
@@ -72,7 +72,8 @@ public enum Authorities {
   F_TRACKED_ENTITY_MERGE,
   F_DATAVALUE_ADD,
   F_IMPERSONATE_USER,
-  F_SYSTEM_SETTING;
+  F_SYSTEM_SETTING,
+  F_MAP_EXTERNAL_LAYER_ADD;
```

Comment:
Could be duplicated with [ExternalMapLayerSchemaDescriptor](https://github.com/dhis2/dhis2-core/blob/7b45170f41767beda521a96fca9a81e3b614c221/dhis-2/dhis-services/dhis-service-schema/src/main/java/org/hisp/dhis/schema/descriptors/ExternalMapLayerSchemaDescriptor.java#L48)